app/src/test/java/org/fossasia/susi/ai/helper/helper/UtilsTest.kt

```diff
@@ -0,0 +1,19 @@
+package org.fossasia.susi.ai.helper.helper
+
+import android.util.Patterns
+import junit.framework.Assert.assertTrue
+import org.fossasia.susi.ai.helper.Utils
+import org.fossasia.susi.ai.rest.responses.susi.SkillData
+import org.junit.Test
+
+class UtilsTest {
+
+    val skillData: SkillData = SkillData()
```

Comment:
@iamareebjamal since the SkillData is always parcelable it returns null pointer exception